solutions/usaco-1060.mdx

```diff
@@ -53,5 +53,83 @@ int main(){
 ```
 
 </CPPSection>
+	
+<JavaSection>
+	
+```java
+// Created by Sathvik Chundru with help from Justin Chen
```

Comment:
i mean there's the author page, seems kinda unnecessary

Reply:
k @24justinch

Reply:
is the guy who I credited. If I have to remove his name then at least let his name be immortalized here